recipe/meta.yaml

```diff
@@ -83,12 +83,12 @@ outputs:
     build:
       number: {{ number }}
       # For CUDA 12, the compiler version is hard-coded in the cuda120.yaml migrator
-      skip: true  # [(py < 39) or cuda_compiler_version not in ("11.8", "12.8") or ppc64le]
+      skip: true  # [(py < 39) or cuda_compiler_version not in ("11.8", "12.4", "12.8")]
       script:
         # CuPy default detects CUDA from nvcc, but on Conda-Forge's dockers nvcc lives in a different place...
         # With conda-forge/nvcc-feedstock#58, CUDA_PATH is set correctly
         - export NVCC="$(which nvcc)"                                                # [linux]
-        - export CUPY_NUM_BUILD_JOBS=1                                               # [aarch64 or ppc64le or (linux64 and cuda_compiler == "nvcc")]
+        - export CUPY_NUM_BUILD_JOBS=1
```

Comment:
Trying to address memory issues we have been seeing on CI by limiting parallelism

xref: https://github.com/conda-forge/cupy-feedstock/issues/301#issuecomment-2692500958

Reply:
Cherry-picking this into PR: https://github.com/conda-forge/cupy-feedstock/pull/302